app/components/global/AppBar/AppBar.md

```diff
@@ -3,11 +3,24 @@ Overall navigation bar for the app
 When a user is not logged in:
 
 ```js
-<AppBar />
+<AppBar
+  menuItems={[
+    { label: 'Author guide', link: '/author-guide' },
+    { label: 'Reviewer guide', link: '/reviewer-guide' },
+    { label: 'Contact us', link: '/contact-us' },
+  ]}
+/>
 ```
 
 When a user is logged in:
 
 ```js
-<AppBar user="anyone" />
+<AppBar
+  menuItems={[
+    { label: 'Author guide', link: '/author-guide' },
+    { label: 'Reviewer guide', link: '/reviewer-guide' },
+    { label: 'Contact us', link: '/contact-us' },
+  ]}
+  user="anyone"
```

Comment:
Is it possible for us to pass in a dummy user object, so that the ProfileMenu within the `AppBar` component in styleguidist is functional?

Right now, this doesn't conform to your new PropTypes definition, so we're getting console errors:
![image](https://user-images.githubusercontent.com/15656538/47856856-19333800-dde0-11e8-8747-6284f88150ec.png)

and we're also not seeing the 'Dashboard' link appear.